dashi/rnaseqc/over_time.py

```diff
@@ -51,72 +75,231 @@
     'Proportion UTR Bases',
 )
 
+# The colours that are used in the graphs
 COLOURS = [
     '#1f77b4', '#aec7e8', '#ff7f0e', '#ffbb78', '#2ca02c',
     '#98df8a', '#d62728', '#ff9896', '#9467bd', '#c5b0d5',
     '#8c564b', '#c49c94', '#e377c2', '#f7b6d2', '#7f7f7f',
     '#c7c7c7', '#bcbd22', '#dbdb8d', '#17becf', '#9edae5'
 ]
 
+# The shapes of the points to use in the graph
+# More can be added if the future needs it
+# See https://plot.ly/python/reference/#scatter-marker
+SHAPES = [
+    "circle", "square", "diamond", "cross", "x", "triangle-up", "triangle-down",
+    "triangle-left", "triangle-right", "pentagon", "star",
+]
+
+# Which columns will the data table always have
+DEFAULT_TABLE_COLUMN = [
+    {'name': 'Library', 'id': 'Sample Name'},
+    {'name': 'Project', 'id': 'Study Title'},
+    {'name': 'Run', 'id': 'Sequencer Run Name'},
+    {'name': 'Lane', 'id': 'Lane Number'},
+    {'name': 'Kit', 'id': 'geo_prep_kit'},
+    {'name': 'Library Source Type', 'id': 'geo_library_source_template_type'},
```

Comment:
`Library Source Type` is referred to as `Library Design` in MISO; is there a need to use a different term here?

Reply:
Changed in bce269f